common/test/acceptance/pages/lms/problem.py

```diff
@@ -413,7 +413,7 @@ def is_correct_choice_highlighted(self, correct_choices):
         """
         Check if correct answer/choice highlighted for choice group.
         """
-        xpath = '//fieldset/div[contains(@class, "field")][{0}]/label[contains(@class, "choicegroup_correct")]'
+        xpath = '//fieldset/div[contains(@class, "field")][{0}]/label[contains(@class, "choicegroup_correct")]/span[contains(@class, "status correct")]'  # pylint: disable=line-too-long
```

Comment:
This will now also verify that the span with the correctness indicator is present.